src/protocols/multi_party_ecdsa/gg_2020/state_machine/keygen/rounds.rs

```diff
@@ -273,7 +273,9 @@ impl Round4 {
         let y_sum = tail.iter().fold(head[0], |acc, x| acc + x);
 
         let local_key = LocalKey {
-            keys_additive: self.keys.clone(),
+            paillier_dk: self.keys.dk,
+            paillier_ek: self.keys.ek,
```

Comment:
`ek` is actually redundant because we have it in the paillier vector later.

Reply:
You're right. In theory, `ek` should be derivable from `dk`, but I didn't find conversion method

Reply:
Fixed - now it uses the paillier vector instead